### ISSUE TITLE
error: unsupported operation: can't call foreign function: pipe2

### ISSUE THREAD

**Issue by botika** [rust-lang/miri#2658]
https://github.com/botika/unblock/actions/runs/3412767433/jobs/5678690824#step:4:124

`MIRIFLAGS=-Zmiri-symbolic-alignment-check -Zmiri-disable-isolation -Zmiri-ignore-leaks`

Launch using `std::process::Command` in another thread. All others test in crate runs well.

**Comment by oli-obk** [comment]
This is already tracked in https://github.com/rust-lang/miri/issues/2057

You can use `-Zmiri-panic-on-unsupported` to turn that interpreter halt into a panic handled by the test suite, or you can `#[cfg(not(miri))]` your test to skip it.

**Comment by RalfJung** [comment]
Yeah, `Command` is not supported by Miri (and it seems quite hard to do so).
